.github/scripts/junit.py

```diff
@@ -55,6 +55,9 @@ class TestCase:
     def key(self) -> Tuple[str, str]:
         return self.class_name, self.test_name
 
+    def __repr__(self):
+        return f"{self.class_name} {self.test_name}"
```

Comment:
Unrelated to this PR, but was causing issues in some cases where a test stacktrace was really long